thoth/common/openshift.py

```diff
@@ -1633,7 +1633,7 @@ def get_adviser_template(self) -> Dict[str, Any]:
                 "Infra namespace is required to gather adviser template when running it"
             )
 
-        return self._get_template("template=adviser")
+        return self._get_template("template=adviser-argo")
```

Comment:
`if self.use_argo`?

Reply:
right, `adviser-workload-operator`

Reply:
We can switch the other way around, as Argo will be the default, as suggested by @goern. WDYT?

Reply:
I think we missed each other:

```python
if self.use_argo:
        return self._get_template("template=adviser")
else:
        return self._get_template("template=adviser-workload-operator")
```

To stay compatible with the env variable switch.

Reply:
Sorry, I didn't get that indeed :) Let's discuss it on Monday :)

Reply:
Updated, as discussed. Thanks! :beers: